### PR TITLE
Fallout Stuff

### PR DESCRIPTION
**Pull request by NigglyWiggly**
Bunch of Fallout related stuff.
New version of bottle Nuka Cola, not the homemade bar drink (though
they're the same reagent), Nuka Cola Quantum, a vending machine that
sells them both, Vault 13 and General Vault-Tec Uniforms, and a 'Booze
Bunker' bar sign.

After this, I'm gonna start working on the Pip Boy 2000 PDA alternative,
which'll probably just be random space loot

### CODE REVIEW

**Comment by NigglyWiggly** [comment]
Oh wait, that quantum reagent thing probably doesn't actually do anything, lemme fix that real quick

**Comment by Nienhaus** [comment]
Here. http://puu.sh/iiYcU/4b13289b2b.dmi I also have a couch and pool table.

**Comment by Kurfursten** [comment]
In your testing, how irradiated does someone get from one bottle of quantum?

**Comment by NigglyWiggly** [comment]
Just a bit more than with uranium

**Comment by Kurfursten** [comment]
How much radiation does 30u uranium put into someone?

**Comment by NigglyWiggly** [comment]
Probably around 3u of mutagen worth

**Comment by FudgePucker** [comment]
Wait uranium isn't even that effective of a toxin?
It's only obtainable from the roid, why is it weaker than something out of the chemical desp?

**Comment by ComicIronic** [comment]
Because uranium's main use isn't as a poison, it's a material.

**Comment by NigglyWiggly** [comment]
Probably gonna need assistance with the Pip boy thing, so I'm gonna make that a whole different PR

**Comment by 9600bauds** [comment]
Uranium is actually not that radioactive.

**Comment by ComicIronic** [comment]
While you're doing the editing, could you fix the typo on nuka_cola that stops it applying life effects?

**Comment by NigglyWiggly** [comment]
Sure thing

**Comment by ririchiyo** [comment]
Please fix up the relative pathing in some of the files.

**Comment by 9600bauds** [comment]
New()? Near damn ALL food items are like that.
Take a peak at snacks.dm.

**Comment by NigglyWiggly** [comment]
So is there actually a problem with the Travis thing, or is Riri just giving me more reasons to tell him he's a butt?

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 1]
```diff
@@ -666,6 +666,28 @@
...
+ /obj/item/weapon/reagent_containers/food/drinks/soda_cans/nuka/New()
+	..()
+	reagents.add_reagent("nuka_cola", 30)
+	src.pixel_x = rand(-10.0, 10)
+	src.pixel_y = rand(-10.0, 10)
+
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/quantum
+	name = "Nuka Cola Quantum"
+	desc = "Take the leap... enjoy a Quantum!"
+	icon_state = "quantum"
```
Comment (clusterfack) [inline code/modules/reagents/reagent_containers/food/drinks.dm]: Need an extra line here

**Comment by NigglyWiggly** [comment]
So it only started fucking up the travis stuff after I changed the pathing, would anyone other than Riri have an objection to if I just reverted back to before then?

**Comment by NigglyWiggly** [comment]
Relative pathing schmelative pathing merge it anyway bby

**Comment by 9600bauds** [comment]
What's so bad anyways? Every food item has a new() since it's the glorified reagent list. That single harmless bit of relative pathing keeps one food item per leftmost tabulation and makes the file actually easier to read.
Not like anyone will ctrl+f "soda_cans/nuka/new()" anyways.

**Pull request merged**